src/coreclr/jit/instrsxarch.h

```diff
@@ -594,8 +594,8 @@ INST3(LAST_AVXVNNI_INSTRUCTION, "LAST_AVXVNNI_INSTRUCTION", IUM_WR, BAD_CODE, BA
 // BMI1
 INST3(FIRST_BMI_INSTRUCTION, "FIRST_BMI_INSTRUCTION", IUM_WR, BAD_CODE, BAD_CODE, BAD_CODE, INS_FLAGS_None)
 INST3(andn,             "andn",             IUM_WR, BAD_CODE,     BAD_CODE,     SSE38(0xF2),                             Resets_OF      | Writes_SF     | Writes_ZF     | Undefined_AF  | Undefined_PF  | Resets_CF     | INS_Flags_IsDstDstSrcAVXInstruction)    // Logical AND NOT
-INST3(blsi,             "blsi",             IUM_WR, BAD_CODE,     BAD_CODE,     SSE38(0xF3),                             Resets_OF      | Writes_SF     | Writes_ZF     | Undefined_AF  | Undefined_PF  | Writes_CF     | INS_Flags_IsDstDstSrcAVXInstruction)    // Extract Lowest Set Isolated Bit
-INST3(blsmsk,           "blsmsk",           IUM_WR, BAD_CODE,     BAD_CODE,     SSE38(0xF3),                             INS_Flags_IsDstDstSrcAVXInstruction)    // Get Mask Up to Lowest Set Bit
+INST3(blsi,				"blsi",				IUM_WR, BAD_CODE,	  BAD_CODE,		SSE38(0xF3),							 Resets_OF		| Writes_SF		| Writes_ZF		| Undefined_AF	| Undefined_PF	| Writes_CF		| INS_Flags_IsDstDstSrcAVXInstruction)    // Extract Lowest Set Isolated Bit
+INST3(blsmsk,			"blsmsk",			IUM_WR, BAD_CODE,	  BAD_CODE,		SSE38(0xF3),							 Resets_OF		| Writes_SF		| Writes_ZF		| Undefined_AF	| Undefined_PF	| Writes_CF		| INS_Flags_IsDstDstSrcAVXInstruction)    // Get Mask Up to Lowest Set Bit
```

Comment:
Are you sure Writes_ZF is correct? shouldn't it be Resets_ZF?

Reply:
`ZF and OF flags are cleared` so it means it has to be Resets_ZF rather than Writes_ZF, right?

Reply:
I'm not sue i'm qualified to answer. The manual lists the pseudo code for the operation as:

```
temp ← (SRC-1) bitwiseAND ( SRC );
SF ← temp[OperandSize -1];
ZF ← (temp = 0);
IF SRC = 0
CF ← 1;
ELSE
CF ← 0;
FI
DEST ← temp;
```
which to me writes to SF ZF and CF, additionally it says that OF is cleared which I read as being reset.

Reply:
so is ZF, even from your pseudocode it looks like it's always cleared
And the doc says `ZF and OF flags are cleared.`

Reply:
Ok, changed.